src/packages/recompose/componentFromStream.js

```diff
@@ -23,17 +32,16 @@ const createComponent = propsToVdom =>
 
     componentWillMount() {
       // Subscribe to child prop changes so we know when to re-render
-      this.subscription = this.vdom$.subscribe(
-        vdom => {
+      this.subscription = this.vdom$.subscribe({
+        next: vdom => {
           this.didReceiveVdom = true
           if (!this.componentHasMounted) {
             this.state = { vdom }
```

Comment:
Why check on hasMounted? (*It was a bug in react, but it solved*)
Also looks like `this.state = {}` could be changed on `this.setState` as this method doe not called from contructor